.docker/entrypoint/lib/commander.rb

```diff
@@ -62,21 +70,32 @@ def execute_command(jekyll_config, command, dry_run, verify)
       end
 
       def verify(jekyll_config)
-        verifier = Verifier.new(jekyll_config)
+        verifier = provide_instance(:verify, jekyll_config)
         verifier.verify
       end
 
       def deploy(jekyll_config, dry_run, verify)
-        deployer = Deployer.new(jekyll_config, @jekyll_var_dir)
+        deployer = provide_instance(:deploy, jekyll_config, @jekyll_var_dir)
         deployer.deploy(dry_run, verify)
       end
 
       def jekyll_command(jekyll_config, command, dry_run)
         log(:warn, "Warning: --dry-run has no effect on the `jekyll #{command}` command.") if dry_run
 
-        jekyll_commander = JekyllCommander.new(jekyll_config)
+        jekyll_commander = provide_instance(command, jekyll_config)
+        # jekyll_commander = Jekyll::PlantUml::Commands::JekyllCommander.new(jekyll_config)
```

Comment:
Codacy found an issue: [Line is too long. [91/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5780130)